ra2ce/graph/networks.py

```diff
@@ -202,6 +220,75 @@ def network_shp(
         # Exporting complex graph because the shapefile should be kept the same as much as possible.
         return graph_complex, edges_complex
 
+    def network_cleanshp(
```

Comment:
```suggestion
    def network_clean_shp(
```

Reply:
Here I considered cleanshp as one type of input file hence the network `network_cleanshp` (create network from shapefile without cleanup, in contrast to create network from shapefile with cleanup - `network_shp`). I agree it is not clear but I prefer to sort that out once we have a cleanup function ready. then all network creation from shapefile can use the same function.

Reply:
In my experience leaving things for later means they won't be picked up. Better to give a proper name now.
Based on the code I'm reading you could emulate the same approach as Sahand and create a separate file / class `NetworkShpWrapper` or similar, where you do this network_clean_shp and any other related, specific, methods required for importing network and graph from a `shp` file. This approach:
- Increases maintainability.
- Applies the `Single Responsibility Principle`.
- Increases testing options (you no longer have to initialize a `Network` object to see the whole import from `shp`.
- Reduces code density in networks.py (as preparation for the upcoming refactoring).

Reply:
'VectorNetworkWrapper' is in progress